scripts/translation/configure.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ *  +----------------------------------------------------------------------+
+ *  | Copyright (c) 1997-2023 The PHP Group                                |
+ *  +----------------------------------------------------------------------+
+ *  | This source file is subject to version 3.01 of the PHP license,      |
+ *  | that is bundled with this package in the file LICENSE, and is        |
+ *  | available through the world-wide-web at the following url:           |
+ *  | https://www.php.net/license/3_01.txt.                                |
+ *  | If you did not receive a copy of the PHP license and are unable to   |
+ *  | obtain it through the world-wide-web, please send a note to          |
+ *  | license@php.net, so we can mail you a copy immediately.              |
+ *  +----------------------------------------------------------------------+
+ *  | Authors:     André L F S Bacci <ae php.net>                          |
+ *  +----------------------------------------------------------------------+
+ *  | Description: Generate cached data for revcheck and QA tools.         |
+ *  +----------------------------------------------------------------------+
+ */
+
+require_once __DIR__ . '/lib/all.php';
+
+if ( count( $argv ) < 2 )
+{
+    fwrite( STDERR , "  Missing paramater. Usage:\n" );
+    fwrite( STDERR , "    {$argv[0]} [lang_dir]:\n" );
+    return;
+}
+
+new RevcheckRun( 'en' , $argv[1] , true );
```

Comment:
Maybe allow a way to provide a custom location to ``en`` ?

Reply:
It's easy to change this configure to accept one or two path locations, but I only tested running all other code from the parent directory of both `en` and `$LANG`. Running from other locations may surface new bugs.

If the server machines don't `cd` or `pushd` into a common checkout directory, or uses other names for `en`, I may work on after the new year.

Reply:
Ack, move as a future TODO